src/ADAL.PCL.iOS/TokenCachePlugin.cs

```diff
@@ -33,6 +33,12 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
     internal class TokenCachePlugin : ITokenCachePlugin
     {
+#if MAC
```

Comment:
in native library we do not persist the cache in keychain when using Mac. the persistence is completely owned by the developer by implemeting the delegates